.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,24 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+    branches: [master, dev]
```

Comment:
`develop`?

Reply:
Nice catch.. Interesting that it ran either-way 🤔

Reply:
Pattern matching https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-branches

Reply:
Added an if statement to the deploy-keeper workflow to ensure we only deploy when on `develop` or `master`